service/gerrit/change_review.go

```diff
@@ -60,6 +60,7 @@ func (g *ChangeReviewCommenter) Post(_ context.Context, c *reviewdog.Comment) er
 // Flush posts comments which has not been posted yet.
 func (g *ChangeReviewCommenter) Flush(ctx context.Context) error {
 	g.muComments.Lock()
+	defer func() { g.postComments = nil }()
 	defer g.muComments.Unlock()
```

Comment:
same as https://github.com/reviewdog/reviewdog/pull/1867#discussion_r1703101026

```suggestion
	g.muComments.Lock()
	defer g.muComments.Unlock()
	defer func() { g.postComments = nil }()
```